weave/trace_server/environment.py

```diff
@@ -24,3 +25,46 @@ def wf_clickhouse_pass() -> str:
 def wf_clickhouse_database() -> str:
     """The name of the clickhouse database."""
     return os.environ.get("WF_CLICKHOUSE_DATABASE", "default")
+
+
+# BYOB Settings
+
+
+def wf_file_storage_uri() -> Optional[str]:
+    """The storage bucket URI."""
+    return os.environ.get("WF_FILE_STORAGE_URI")
+
+
+def wf_storage_bucket_aws_access_key_id() -> Optional[str]:
+    """The AWS access key ID."""
+    return os.environ.get("WF_FILE_STORAGE_BUCKET_AWS_ACCESS_KEY_ID")
```

Comment:
Are we sure we want to use `WF_*` env vars?  How will a user know what this means?

Reply:
again, this is not user facing, it is our internal deployment... but i agree we might want to break the WF_* pattern